SwiftWisdom/Core/StandardLibrary/Numbers/Int+Extensions.swift

```diff
@@ -65,3 +65,11 @@ extension NSNumberFormatter {
         return decimalFormatter
     }
 }
+
+public extension Int {
```

Comment:
That's a good point about using NSTimeInterval + NSDateFormatter. I'll just remove that function and corresponding test.